File: bee_3060.py
def main():
    valorDaCompra = int(input())
    numParcelas = int(input())

    resto = valorDaCompra % numParcelas

    for i in range(numParcelas):
        thisResto = int(resto > 0)
        print(int(valorDaCompra / numParcelas) + thisResto)
        resto -= 1

main()

File: bee_3060.c
#include <stdio.h>

int main() {
    int valorDaCompra, numParcelas;
    scanf("%d", &valorDaCompra);
    scanf("%d", &numParcelas);

    int resto = valorDaCompra % numParcelas;
    int i;

    for (i = 0; i < numParcelas; i++) {
        int thisResto = (resto > 0) ? 1 : 0;
        printf("%d\n", (valorDaCompra / numParcelas) + thisResto);
        resto--;;
    }

    return 0;
}
